indexer/services/comlink/src/controllers/api/v4/vault-controller.ts

```diff
@@ -74,7 +105,121 @@ class VaultController extends Controller {
       .value();
 
     return {
-      vaultsPnl: groupedVaultPnlTicks,
+      vaultsPnl: _.sortBy(groupedVaultPnlTicks, 'ticker'),
+    };
+  }
+
+  @Get('/megavault/positions')
+  async getMegavaultPositions(): Promise<MegavaultPositionResponse> {
```

Comment:
this doesn't yet include `megavault` module account subaccoun 0? Is that planned for the future

Reply:
No, that is not part of the requirements of the endpoint as of now.